WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -550,7 +550,7 @@ public void onMediaChanged(MediaStore.OnMediaChanged event) {
 
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
-    public void OnMediaListFetched(MediaStore.OnMediaListFetched event) {
+    public void onMediaListFetched(MediaStore.OnMediaListFetched event) {
```

Comment:
Note: I made `OnChanged` events `static` in https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/338.